pyo3-macros-backend/src/method.rs

```diff
@@ -199,8 +199,8 @@ impl CallingConvention {
         } else if accept_kwargs {
             // for functions that accept **kwargs, always prefer varargs
             Self::Varargs
-        } else if cfg!(not(feature = "abi3")) {
-            // Not available in the Stable ABI as of Python 3.10
+        } else if cfg!(not(feature = "abi3-py39")) {
```

Comment:
What about 3.7 and 3.8?

Reply:
`abi3-py37` and `abi3-py38` imply `abi3-py39`, `abi3-py310` and `abi3`.

`abi3-*` features work backwards compared to `Py_3_*` configs.